.github/workflows/license_check.yaml

```diff
@@ -21,5 +21,8 @@ jobs:
     name: License Check
     steps:
       - uses: actions/checkout@v4
-      - run: cargo install --locked cargo-deny
+      - name: Install cargo deny
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-deny@0.15.1
```

Comment:
we were installing the latest version of cargo-deny, the format has since changed and broke CI.
Quickly fix the issue by installing a specific version.